src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/utils/Common.java

```diff
@@ -113,5 +120,10 @@ public void updateTaskWithCustomVariablesOverride(Map<String,String> task,
 
     }
 
+    public void cleanUpTask(String taskId) {
+        camundaApiActions.post(ENDPOINT_COMPLETE_TASK, taskId,
+                               authorizationHeadersProvider.getServiceAuthorizationHeadersOnly());
+    }
```

Comment:
Can we check the state of the completed task? I.e. check that the task no longer exists as part of completing and clean up.